Лабораторная работа 1/Task_1.py

```diff
@@ -0,0 +1,15 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+skip1 = numbers[5:]
+skip2 = numbers[:4]
+summ1 = sum(skip1)
+lenn = len(numbers)
+avarage1 = round(summ1/lenn)
+summ2 = sum(skip2)
+avarage2 = round(summ2/lenn)
+sum_so = summ1 + summ2
```

Comment:
Данный фрагмент кода можно выполнить более корректно и в сокращенной версии
```suggestion
avarage = round(sum(numbers[:4] + numbers[5:]) / len(numbers))
```